docs/source/markdown/options/device-read-bps.md

```diff
@@ -0,0 +1,5 @@
+#### **--device-read-bps**=*path:rate*
+
+Limit read rate (in bytes per second) from a device (e.g. **--device-read-bps=/dev/sda:1mb**).
+
+This option is not supported on cgroups V1 rootless systems.
```

Comment:
(I thought I had made this comment yesterday, but can't find it now).

This option does not seem to work rootless in cgroups v2, either. In fact most options don't. Should it (and the others) just say `rootless`, and remove the `cgroups v1`? This is not a blocker; it might need to be reevaluated later.

Reply:
(Addressed in #15565)